frontend/src/pull-card.tsx

```diff
@@ -1,26 +1,29 @@
 import { Pull } from './pull';
-import ListGroup from 'react-bootstrap/ListGroup';
-import { CommitStatuses, Status } from './commit-statuses';
-import styled from 'styled-components';
+import { CommitStatuses } from './commit-statuses';
+import { Flex, Box, Link, useStyleConfig, chakra } from "@chakra-ui/react"
 
-const PullCardContainer = styled(ListGroup.Item)`
-   border-left: 0;
-   border-right: 0;
-
-   &:hover ${Status} {
-      display: block;
+const Card = chakra(Flex, {
+   baseStyle: {
+      p: 2,
+      pl: 0,
+      borderTop: "1px #ccc solid",
+      "&:hover .build_status": {
+         opacity: 1
+      },
    }
-`;
+});
```

Comment:
My understanding is that the [`chakra()` ](https://chakra-ui.com/docs/features/chakra-factory#chakra-factory-function)factory function is primarily used to pass style props to non-chakra components. So using `chakra(ChakraComponent, { ... })` might not be the pattern we want to use here, unless there's something special about this styling that I'm missing.

Reply:
I was experimenting with different forms of styling, having all this styling in the JSX below obscures the structure. I don't think we should exclude a form of styling unless there's a reason to.

Reply:
Got it 👍🏻  makes sense